Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,41 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            string name = "Hancock";
```

Comment:
Input should not be hard coded. Use Console.ReadLine(...) to populate these values and then convert them to the appropriate types.

Reply:
I misunderstood Step 3, I saw it as making an entire new set of values. The readline was there to bypass a issue that made my code close when I ran it. Figured out what was the issue so Ill adjust the code to allow user input.

Reply:
Any update on this? Not approving until you read user input into your variables.

Reply:
I believe I did it correctly